README.md

```diff
@@ -45,6 +45,19 @@ See below for steps which are required for installation.
 
 See below for brief tutorials on how to implement the work found in this repository for your needs.
 
+### 🎛️ State Management
+
+These steps cover how to store [Terraform state management](https://developer.hashicorp.com/terraform/language/state) files in association with the bucket infrastructure below.
+State management is used to share access to maintenance files required after resources are created by those in the role of Terraform Administrators.
+This repository assumes Terraform state is managed separately from other work within the same Google Cloud project.
```

Comment:
While this is true in this case, I'm not sure I'd want to draw attention to it here. One, because I'd personally like that to change (IMHO each project should be for a specific purpose, but I suppose we need OIT GCP for that) and two because I think it might be out of scope for the reader of the tutorial.